actionview/lib/action_view/helpers/tags/collection_check_boxes.rb

```diff
@@ -25,7 +25,7 @@ def render(&block)
 
         private
           def render_component(builder)
-            builder.checkbox + builder.label
+            builder.checkbox.to_s + builder.label
```

Comment:
We should define `+` in the `Tags::Base` to make sure we an concatenate two tags without having to call `to_s`.

Reply:
Nice point! With the `+` method, I had to change even fewer tests.